proto/firecracker.proto

```diff
@@ -26,6 +26,20 @@ message VMConfiguration {
   // TODO: add container_image here?
 }
 
+message SnapshotKeySet {
+  // Snapshot key to be read from when resuming the snapshot and written to when
+  // the task is complete.
+  //
+  // For workflows, this is associated with the git branch checked out in the
+  // snapshotted git repo directory.
+  SnapshotKey branch_key = 1;
```

Comment:
Personally I prefer the language of something like `primary_key`. It's harder for me to conceptualize what a `branch_key` is

Reply:
haha, I originally discussed this with Tyler offline and we found `main_key` / `primary_key` to also be confusing and settled on `branch_key` :sweat_smile: 

I think this name is not super consequential so would prefer to leave as-is for now.